graylog2-server/src/main/java/org/graylog/security/certutil/keystore/storage/location/KeystoreMongoCollections.java

```diff
@@ -27,8 +27,15 @@ public interface KeystoreMongoCollections {
             "encrypted_certificate_keystore"
     );
 
+    KeystoreMongoCollection GRAYLOG_CA_KEYSTORE_COLLECTION = new KeystoreMongoCollection(
+            "graylog_ca_certificates",
+            "node_id",
+            "encrypted_certificate_keystore"
+    );
```

Comment:
(Minor) I'd vote for some nice, common prefix to our cert-related collections, so that they appear close to each other in Mongo.